### ISSUE TITLE
Fmask4.0 Standalone without GUI not running on Ubuntu

### ISSUE THREAD

**Issue by ghost** [GERSL/Fmask4#6]
Hi,

I have installed the Fmask4.0 Standalone on Ubuntu 14.04.

However, when trying to run the Software I receive the following error:

error while loading shared libraries: libmwlaunchermain.so: cannot open shared object file: No such file or directory.

This appears to be a common error across other applications like in the below link:

https://stackoverflow.com/questions/24004240/matlab-2014a-8-3-compiler-runtime-errors-libmwlaunchermain-so

However, following these instructions, I haven't solved the issue. I'd be really grateful for some help in configuring everything correctly! 

Thanks!

**Comment by j08lue** [comment]
Same here on Ubuntu 18.04

```
Processing granule L1C_T19LCD_A016867_20180914T145042
------------------------------------------
Setting up environment variables
---
LD_LIBRARY_PATH is .:/usr/local/MATLAB/MATLAB_Runtime/v93/runtime/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/v93/bin/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/v93/sys/os/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/v93/sys/opengl/lib/glnxa64
/usr/GERS/Fmask_4_0/application/Fmask_4_0: error while loading shared libraries: libmwlaunchermain.so: cannot open shared object file: No such file or directory
```

**Comment by j08lue** [comment]
@willray3 did you ever find a solution?

**Comment by j08lue** [comment]
Right, the origin of this problem for me was just that the runtime is now `v95` instead of `v93` and I needed to fix my `MCROOT` variable (set to `/usr/local/MATLAB/MATLAB_Runtime/v95`). When that variable is set (correctly), the MATLAB runtime apparently sets the `LD_LIBRARY_PATH` etc correctly for you.

Problem solved for me and I'd not be surprised if it was a similarly easy fix for you @willray3.
